src/countOccurrence.md

```diff
@@ -51,3 +72,34 @@ function countOccurrences($arr, $valueToCount) {
     return $count;
 }
 ```
+
+### Rust
+
+```rust
+fn count_occurrences<A: Eq>(arr: Vec<A>, value_to_count: A) -> usize {
+    let mut count: usize = 0;
+    for element in arr {
+        if element == value_to_count {
+            count += 1;
+        }
+    }
+    return count;
+}
+```
+
+### CPP
+
+```cpp
+#include <vector>
```

Comment:
See if you can do it without using any lib/dependancy so the function can be used easily
Just like other snips, so easily can be used